### PR TITLE
Document the process for release branches

### PR DESCRIPTION
**Pull request by eregon**
* Fix "jt pr" to not rename the branch for release branches.

### CODE REVIEW

**Comment by eregon** [comment]
@chrisseaton updated.
@ansalond Please review this PR.

File: doc/contributor/release-branches.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Releases Branches
+
+Binary suites are needed to run the GraalVM gate.
+Therefore, some care must be taken so they are deployed.
+
+### If there is no release branch in GraalVM
+
+Just make a PR against `master` as usual.
+The latest commit from TruffleRuby master will be picked up when creating the
```
Comment (wirthi) [inline doc/contributor/release-branches.md]: (just asking) when do you put a linebreak? For textfiles I usually put one whole sentence on one line.

Comment (chrisseaton) [reply doc/contributor/release-branches.md]: So it's pleasantly readable as a plain text file.

**Comment by eregon** [comment]
@chrisseaton @nirvdrum @pitr-ch @aardvark179 When making a fix to go in GraalVM, please read this documentation and check with the release manager.

**Pull request merged**